packages/template-retail-react-app/app/components/item-variant/item-price.jsx

```diff
@@ -95,6 +110,30 @@ const ItemPrice = ({currency, align = 'right', baseDirection = 'column', ...prop
             {isDesktop && variant?.quantity > 1 && !isASet && priceData?.pricePerUnit && (
                 <PricePerItem currency={currency} basePrice={priceData.pricePerUnit} />
             )}
+
+            {/*For screen reader, we want to make sure the product name is announceed*/}
+            <VisuallyHidden role="status">
```

Comment:
Note: we can put this one into the DisplayPrice because the screen reader will only announce the price without mentioning the product name which can cause confusion for listeners. We deliberate create the supported text here to make sure the product name is always said before the price update

Reply:
Is this because DisplayPrice does not have name as the input property? If that is the only reason I think it is worth considering adding the prop. This will avoid calculating the price at two places

Reply:
That is one of the reason, but another reason is the display price is used in different places that is not considered live region, but the screen reader may read the price out lout when it is re-rendered. This code here will. make sure only product-item will get announced on cart item changes

Reply:
can we add a name and and boolean flag for live region(default is false) to DisplayPrice? This way logic will be at one place and code is easy to understand/cleaner

Reply:
thanks for the feedback. I've addressed them as suggested.